modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/Args.kt

```diff
@@ -86,8 +86,8 @@ class Args {
     @Parameter(names = arrayOf("--projectInfo"), description = "Display information about the current projects")
     var projectInfo: Boolean = false
 
-    @Parameter(names = arrayOf("--noIncrementalKotlin"), description = "Disable incremental Kotlin compilation")
```

Comment:
Note that this is a breaking change since you are renaming a flag.

I'm okay with it, but the documentation should be updated as well.

Reply:
I agree that the best solution would be to figure out how the new Gradle does it and fix incremental compilation, but I'm not sure I'll have time for that soon.  

On my projects, the new compiler (1.2.70) without incremental compilation runs faster than the old one with incremental.